rocAL/rocAL/source/rali_api_meta_data.cpp

```diff
@@ -133,6 +133,17 @@ RALI_API_CALL raliCreateTFReaderDetection(RaliContext p_context, const char* sou
     return context->master_graph->create_tf_record_meta_data_reader(source_path , MetaDataReaderType::TF_DETECTION_META_DATA_READER,  MetaDataType::BoundingBox, feature_key_map);
 }
 
+RaliMetaData
+RALI_API_CALL raliCreateMXNetReader(RaliContext p_context, const char* source_path, bool is_output)
+{
+    if (!p_context)
+        THROW("Invalid rali context passed to raliCreateMXNetReader")
```

Comment:
can't use THROW here because there is no try catch block. Return error instead

Reply:
Done